tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27-pytest4-{linux,windows}, py{36,37,38,39}-pytest{4,5,6}-{linux,windows}
+envlist = py27-pytest4-{linux,windows}, py{36,37,38,39,310}-pytest{4,5,6}-{linux,windows}
```

We have to add `pytest7` env for envlist to test with pytest 7.
```suggestion
envlist = py27-pytest4-{linux,windows}, py{36,37,38,39}-pytest{4,5,6,7}-{linux,windows}, py{310}-pytest{6,7}-{linux,windows}
```

---

This is just to run if nothing was specified, right? You can request an environment that's not in the env list manually, like we are doing in CI. (still needs to be added, but it would have been running without this in CI, except for the mistake below, which was causing it to run pytest6 again)